tests/lcs/core/engine/extension.livecodescript

```diff
@@ -161,3 +161,19 @@ on TestExtensionLibraryHandlerCallErrors
          the long id of me, \
          886
 end TestExtensionLibraryHandlerCallErrors
+
+//////////
+
+on TestExtensionLibraryDelimiterAccess
+   set the itemDelimiter to "a"
+   set the lineDelimiter to "b"
+   set the columnDelimiter to "c"
+   set the rowDelimiter to "d"
+
+   local tDelimiters
+   put TestExtensionDelimiterAccess() into tDelimiters
+   combine tDelimiters with comma
+
+   TestDiagnostic tDelimiters
+   TestAssert "library handler can access delimiters", tDelimiters is "a,b,c,d"
+end TestExtensionLibraryDelimiterAccess
```

Comment:
Not at present no - there's no easy way to get LCB code to execute 'out of context' - it would need to be a handler which runs directly from the invocation of the root event loop.